README.md

```diff
@@ -8,16 +8,30 @@
 
 ---
 
-**ProbNum implements probabilistic numerical methods in Python.** Such methods solve numerical problems from linear
-algebra, optimization, quadrature and differential equations using _probabilistic inference_. This approach captures 
-uncertainty arising from _finite computational resources_ and _stochastic input_. 
+ProbNum is a Python toolkit for solving numerical problems in linear algebra, optimization, quadrature and
+differential equations. ProbNum solvers not only estimate the solution of the numerical problem, but also its
+uncertainty (the error) which certainly arises from finite computational resources, discretization, and stochastic input.
 
----
+Currently, available solvers are:
+
+- **Linear solvers:** Solve *Ax = b* for *x*.
+- **ODE solvers:** Solve *&#7823;(t) = f(&#8201;y(t), t&#8201;)* for *y*.
+- **Quadrature solvers:** Solve *F = &#x222b; f(x)&#8201;p(x)&#8201;dx* for *F*.
```

Comment:
**Quadrature routines**

Reply:
I was thinking about this, too, as it sounded unfamiliar. But I couldn't find a reason not to use solvers analogous to the other solvers. After all, quadrature methods solve the integration problem for the internal value *F* from incomplete data.

Reply:
I certainly agree with the last sentence. What sounded strange to me is that quadrature routines do not solve quadrature as ODE solvers solve ODEs. Rather quadrature already is the solution process, i.e. "solving an equation by quadrature". With the same logic integral solvers is much more sensible, but not really well known language.

Reply:
that makes total sense. what would you prefer to go for? "quadrature routines" or "integral solvers"? I wouldn't mind either.

Reply:
Let's go with integral solvers. I think integration is actually probably more understandable to the layperson than quadrature.

Reply:
I went for "Integral solvers (quadrature):" now.

Reply:
oups, sorry did not see your comment as page wasn't updated. Will leave it at "Integral solvers (quadrature):" then.